prusti-tests/tests/verify/pass/arrays/array-mut-loop.rs

```diff
@@ -0,0 +1,19 @@
+use prusti_contracts::*;
+
+fn main() {}
+
+fn looping() {
+    let mut a = [0; 3];
+
+    let mut i = 0;
+    let mut cont = true;
+    while cont {
```

Comment:
Why this strange `cont` boolean instead of `i < 3`?

Reply:
somehow didn't think that's supported. updated. i can't find the documentation that says what loop conditions are supported/unsupported right now. :-/

Reply:
I think at this point we support basically arbitrary code in the loop guard? We don't support e.g. `for` loops, but IIUC it's because of the iterator there.